### ISSUE TITLE
Mist cant connect to any peers

### ISSUE THREAD

**Issue by filiprashev** [ethereum/mist#3492]
Hello I had used mist for some time, but yesterday after windows installed some updates (assuming that could be a problem) I could no longer connect to any peers.
I tryed disabling the firewall, but probem is still there.
![mist problem](https://user-images.githubusercontent.com/32187124/34453202-773b0318-ed56-11e7-99d6-17fe255054d3.png)
I got this problem, prior mist stopped connecting to any peers.

I also use to see the ETH balance I had, now I see no balance at all.
```
Version: `0.9.3`
OS & Version: windows 10
get 1,7,0
latest block sync: 4,368,588
`

**Comment by filiprashev** [comment]
I managed to do some port opening and firellow setting. Im not getting 1 peer and back to 0 and agan 1 and back to 0. This is not good. Any help pls?

**Comment by filiprashev** [comment]
I did lots of things and even del the geth folder, trying to sync all over again. Still nothing and fail to find any peers.

**Comment by filiprashev** [comment]
Now both ETH wallet and Mist are syncing simultaneously with the same speed... so after 24h and more and several restarts, ETH wallet stopped syncing and Mist countiniue for a little while. Eventually slow down the process and stopped syncing..
I decided to delete and uninstall all of ETH wallet and mist data and start all over again.

So after re-install I managed to get Mist running on its own without the ETH wallet. Up to about 1m block left.. and again stop and restart, stop and restart... many times.. 

I tried the loght network, at first it didint want to find any peers... then after waiting for a quite a long time it found 1 peer and start syncing.. up to about 1m block and stopped...
